### PR TITLE
Fix for issue #20

### LINKED ISSUE #20: Swift version
Is Swift version planned?

### PR DESCRIPTION
**Pull request by michaelkruglos**
## TL;DR
This change along with #22, fixes #20

## Long Story

Issue #22 solves the case, when you integrate existing project with react-native, and want to use CocoaPods for react-native-branch for iOS.

The error about non-modular headers is still there, when trying to use the code from Swift files.
There's a blog post, which explains non-modular headers issue in cocoapods, under the section "Common Header Pitfalls": http://blog.cocoapods.org/Pod-Authors-Guide-to-CocoaPods-Frameworks/

This problem doesn't appear with Objective C, but does happen in Swift, because swift has a different treatment of modules as well as header files. So when an existing project in Swift gets integrated with react-native and react-native-branch, using cocoapods, the problem above prevents the Swift code from accessing 
RNBranch code, necessary for the initialization under iOS.

### CODE REVIEW

**Comment by rt2zz** [comment]
👍  thank you sir
